src/NDItk/multigroup/OutgoingParticleTransportData/src/verify.hpp

```diff
@@ -0,0 +1,18 @@
+/**
+ *  @brief Verify the weights
```

Comment:
Verify the OutgoingParticleTransportData

Reply:
sigh. I need to start paying attention :-)